api/v1alpha1/nonadminbackup_types.go

```diff
@@ -74,6 +74,68 @@ type VeleroDeleteBackupRequest struct {
 	Namespace string `json:"namespace,omitempty"`
 }
 
+// TODO
+type DataMoverVolumeBackups struct {
+	// number of DataUploads related to this NonAdminBackup's Backup
+	// +optional
+	Total int `json:"total,omitempty"`
+
+	// number of DataUploads related to this NonAdminBackup's Backup in phase New
+	// +optional
+	New int `json:"new,omitempty"`
+
+	// number of DataUploads related to this NonAdminBackup's Backup in phase Accepted
+	// +optional
+	Accepted int `json:"accepted,omitempty"`
+
+	// number of DataUploads related to this NonAdminBackup's Backup in phase Prepared
+	// +optional
+	Prepared int `json:"prepared,omitempty"`
+
+	// number of DataUploads related to this NonAdminBackup's Backup in phase InProgress
+	// +optional
+	InProgress int `json:"inProgress,omitempty"`
+
+	// number of DataUploads related to this NonAdminBackup's Backup in phase Canceling
+	// +optional
+	Canceling int `json:"canceling,omitempty"`
+
+	// number of DataUploads related to this NonAdminBackup's Backup in phase Canceled
+	// +optional
+	Canceled int `json:"canceled,omitempty"`
+
+	// number of DataUploads related to this NonAdminBackup's Backup in phase Failed
+	// +optional
+	Failed int `json:"failed,omitempty"`
+
+	// number of DataUploads related to this NonAdminBackup's Backup in phase Completed
+	// +optional
+	Completed int `json:"completed,omitempty"`
+}
+
+// TODO
```

Comment:
I accept suggestions to field description

Reply:
Just suggestions - no need to include them